hoot-core/src/main/cpp/hoot/core/cmd/AddOsmPbfHeaderCmd.cpp

```diff
@@ -97,6 +97,11 @@ class AddOsmPbfHeaderCmd : public BaseCommand
     *os << is->rdbuf();
     os->flush();
 
+    if (is != &cin)
+      delete is;
+    if (os != &cout)
+      delete os;
+
```

Comment:
When creating a new `fstream` object for `is` and `os` they were never being cleaned up.  We can't just delete them because in some cases they point to `cin` and `cout`.  Probably shouldn't try to `delete` those.